ansible/roles_ocp_workloads/ocp4_workload_le_certs_cert_manager/defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+cert_manager_ns: "cert-mgmt"
+cert_configs_ns: "cert-mgmt"
+cert_manager_helm_ver: "v1.6.1"
```

Comment:
So you don't trust the Red Hat operator either? :-)

Reply:
Kind of a - this PR major goal is to migrate what we've been doing in Open Innovation Labs for some time :)